docs/static/monitoring-apis.asciidoc

```diff
@@ -0,0 +1,312 @@
+[[monitoring]]
+== Monitoring APIs
+
+Logstash provides the following monitoring APIs to retrieve runtime metrics
+about Logstash:
+
+* <<hot-threads-api>>
+* <<root-resource-api>>
+* <<stats-info-api>>
+
+//NOTE: Need to add this to the doc after Alpha 1: * <<plugins-api>>
+
+=== Testing Logstash APIs in Sense
+
+//QUESTION: What do you think of this approach? I felt that the links for invoking Sense would be useful, but I wanted to avoid confusing users. If they don't notice this topic, they might still be confused. Do you think I should keep the links that launch Sense, or should I remove them?
```

Comment:
I think sense is a great idea, but I am not sure it will work out of the box since the target is not an elasticsearch instance and I believe sense uses the kibana configuration to get the address of the server?